src/stopcrit.jl

```diff
@@ -121,3 +121,19 @@ function stop(s::Pereira, stats::AbstractSDDPStats, totalstats::AbstractSDDPStat
         false
     end
 end
+
+
+
+"""
+$(TYPEDEF)
+
+Stops if the gap between the upper and lower bound is lower than ϵ.
+"""
+
+mutable struc Shapiro <: AbstractStoppingCriterion
+    epsilon::Float64
```

Comment:
Should we add a tolerance parameters? (to define 95% or 99% confidence interval)